packages/yourloops/components/statistics/patient-statistics.tsx

```diff
@@ -48,8 +48,8 @@ export const PatientStatistics: FunctionComponent<PropsWithChildren<PatientStati
   const cbgPercentageBarChartData = cbgStatType === CBGStatType.TimeInRange
     ? dataUtil.getTimeInRangeData()
     : dataUtil.getReadingsInRangeData()
-  const { sensorUsage, total } = dataUtil.getSensorUsage()
   const cbgSelect = dataUtil.bgSources.cbg
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (non-blocking):** ‏Could be renamed to `cbgSelected` , it will be more explicit (it's a boolean)